Assets/MRTK/SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -361,6 +361,12 @@ protected virtual void Start()
 
         private void Update()
         {
+            if ((CoreServices.InputSystem == null) ||
```

Comment:
Can you add a comment on this (i.e. designed to avoid running in cases where the MRTK instance is reinitializing)